src/components/BubbleSort.js

```diff
@@ -0,0 +1,79 @@
+export default class BubbleSort {
+  constructor(elements) {
+    this.numberElements = elements;
```

Comment:
Wystarczy this.elements lub this.unsortedElements. Jeśli chcesz opisać typ danych wewnątrz wtedy dodaje się komentarz JSdoc https://jsdoc.app/